config/govuk_index/migrated_formats.yaml

```diff
@@ -209,6 +208,7 @@ migrated:
   - /world/niger
   - /world/nigeria
   - /world/north-korea
+  - /world/macedonia
```

Comment:
Should this be `/world/north-macedonia`?

Reply:
Thanks @thomasleese - careless mistake by me! 🙈